index.js

```diff
@@ -31,7 +54,6 @@ export default class AWSWebsocketProvider extends WebsocketProvider {
     let headers = {
       'Authorization': req.headers['Authorization'],
       'X-Amz-Date': req.headers['X-Amz-Date'],
-      'NodeId': 'nd-ZNOJJ2XNMJAUFP2MSIRL2N45WI',
```

Comment:
Was this in my code? Yikes! Thanks for catching that.

Reply:
no worries, seemed to be a remnant from testing